jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/jobs/HostAssetDataImporter.java

```diff
@@ -118,11 +118,6 @@ public Map<String, Object> execute() {
             procssInfo = fetchHostAssets(type);
         }catch (RuntimeException e)
         {
-            Map<String,String> errorMap = new HashMap<>();
```

Comment:
@AishwaryaKulkarni97 @raviteja-mandala This will completely swallow the error, instead, we just wanted not to raise this as an alert correct?